### DISCUSSION TITLE
Port Tamagotchi Emulator to MicroHydra

### DISCUSSION THREAD

**Discussion by raxp122** [echo-lalia/Cardputer-MicroHydra#64] [Ideas]
I found a cool project for an arduino emulator for the original Tamagotchi [here](https://github.com/GaryZ88/ArduinoGotchi) and it would be awesome if a skilled programmer could make it work on MicroHydra

**Comment by echo-lalia** [comment]
Thanks for sharing! That's an extremely cool looking project and I'd love to mess around with it.

Microhydra apps are natively coded in MicroPython, whereas that project is written in Arduino. So, probably if it gets ported to the Cardputer, it would be an Arduino project released as a .bin file, since you'd otherwise just be rewriting the whole thing. If anyone gives it a try, though, I'll certainly be interested 😁

(It *is* also possible to package compiled code in a .mpy file but I haven't messed around with that).
